pontos/github/api/release.py

```diff
@@ -25,6 +26,97 @@
 
 
 class GitHubRESTReleaseMixin:
+    def create_github_tag(
```

Comment:
IMHO github is superfluous here because the class is already named after GitHub
```suggestion
    def create_tag(
```

Reply:
changed